client/pages/Event/EventDetailsPage.jsx

```diff
@@ -186,46 +206,66 @@ class EventDetailsPage extends React.Component {
   };
 
   handleBack() {
-    const { history: { push } } = this.props;
+    const {
+      history: { push }
+    } = this.props;
     push('/dashboard');
   }
 
   loadEvent() {
-    const { match: { params: { eventId } } } = this.props;
+    const {
+      match: {
+        params: { eventId }
+      }
+    } = this.props;
     const { getEventAction } = this.props;
     getEventAction(eventId);
   }
 
+  showSlackChannels = evt => {
+    evt.preventDefault();
+    this.setState(
+      {
+        showSlackChannels: true
+      },
+      () => {
+        document.addEventListener('click', this.closeChannelsMenu);
+      }
+    );
+  };
+
+  closeChannelsMenu = () => {
+    this.setState({ showSlackChannels: false }, () => {
+      document.removeEventListener('click', this.closeChannelsMenu);
+    });
+  };
   rsvpEvent() {
     const {
       attendEventAction,
-      event: { id },
+      event: { id }
     } = this.props;
     attendEventAction(id);
   }
 
   renderCreateEventButton = eventData => (
     <ModalContextCreator.Consumer>
-      {({
-        activeModal, openModal,
-      }) => {
-        const {
-          categories, uploadImage, updateEvent,
-        } = this.props;
+      {({ activeModal, openModal }) => {
+        const { categories, uploadImage, updateEvent } = this.props;
         if (activeModal) return null;
         return (
           <button
             type="button"
-            onClick={() => openModal('UPDATE_EVENT', {
-              modalHeadline: 'Update event',
-              formMode: 'update',
-              formId: 'event-form',
-              eventData,
-              categories,
-              createEvent: () => '',
-              updateEvent,
-              uploadImage,
-            })
+            onClick={() =>
```

Comment:
This is a general remark for files that you didn't work on but are apart of your commit diff. Please edit your prettier plugin rules to strictly conform to our `stylelint` configuration so that it doesn' t keep formatting every document you open.

Reply:
okay, I'll do that